aal_purchase_form/reports/purchase_order_form.xml

```diff
@@ -0,0 +1,271 @@
+<odoo>
+    <template id="aal_purchase_order_form">
+        <div class="header">
+            <div class="row">
+                <div class="col-5">
+                    <div>
+                        <img src='/aal_purchase_form/static/src/img/aal_logo.jpg' style="max-height: 75px;"/>
+                    </div>
+                    <table class="mt16">
+                        <tr>
+                            <td colspan="2">
+                                25 Soi Phraya Phiren, Chuaploeng Road, Sathon, Bangkok 10120 Thailand
+                            </td>
+                        </tr>
+                        <tr>
+                            <td>T. +662 240 1331-2</td>
+                            <td>F. +662 249 4283</td>
+                        </tr>
+                    </table>
+                </div>
+                <div class="col-7">
+                    <h2 class="text-right">PURCHASE ORDER</h2><br/>
+                    <div class="row">
+                        <div class="col-9 text-right">
+                            <p>
+                                Document No. :<br/>
+                                Date :<br/>
+                                Page :
+                            </p>
+                        </div>
+                        <div class="col-3" t-if="report_type == 'pdf'">
```

Comment:
```suggestion
                        <div class="col-3">
```
Not necessary because we will print pdf only (odoo can show preview on form view)